woocommerce/woocommerce-android
# Issue/2291 print shipping labels

Fixes #2291 by adding the option to print a shipping label.

#### Changes
- Adds new fragment/module/viewmodel to print a shipping label.
- Adds a dialog to display the different paper sizes when printing a shipping label.

#### Screenshots
<img src="https://user-images.githubusercontent.com/22608780/94119742-f21aff00-fe6c-11ea-92f0-10a5e26b0fa4.gif" width="300"/>

#### Testing
- Click on an order from the Orders tab that has shipping labels.
- Notice that if the shipping label is refunded, the `Print shipping label` button is NOT displayed.
- Click on the `Reprint shipping label` button.
- Click on the paper size text field and choose any paper size.
- Click on the `Print shipping label button`.
- Notice the progress bar displayed. A pdf should be opened with a preview of the shipping label.

**Note that both the info options in the Print shipping label screen is not clickable right now but will be taken care of in another issue in #2295** 

Update release notes:
